tower.yml

```diff
@@ -0,0 +1,5 @@
+reports:
+  multiqc_report.html:
+    display: "MultiQC HTML report"
```

Comment:
It would be amazing to have more info here about the expected files

Reply:
Addressed here https://github.com/nf-core/raredisease/pull/346/files